commit 5d611fb49c924c57b9b9a2c21422ee8ac9245564
Author: Trent Mick <trentm@gmail.com>
Date:   2019-03-15T17:32:39-07:00 (7 months ago)
    
    TRITON-884 remove cloud analytics (ca) from Triton (missing ca ref in docs)

diff --git a/docs/config.md b/docs/config.md
index a82ecb8..2196cbd 100644
--- a/docs/config.md
+++ b/docs/config.md
@@ -222,7 +222,6 @@ different than `sdc-headnode` for later review:
 - `sapi_admin_ips`: Only sdc-headnode, likely more important than the others.
 - `papi_admin_ips`: `node-sdc-clients(runtests)` only
 - `napi_admin_ips`: `node-sdc-clients(runtests)` only
-- `ca_admin_ips`: `node-sdc-clients(runtests)` only
 - `dhcpd_admin_ips`: Not used but in `prompt-config.sh`
 - `assets_admin_ips`: `sdc-cnapi`
 - `amon_admin_ips`: `sdc-vmapi`, `sdc-ufds`, `sdc-cloudapi`, `node-sdc-clients(runtests)`, `sdc-amon`
